deployment/1_setup_test_environment.js

```diff
@@ -15,7 +15,7 @@ const BYTES32_NULL = '0x00000000000000000000000000000000000000000000000000000000
 // For development purpose
 async function deployENSRegistry(deployer, owner, domain) {
     // Deploy the public ENS registry
-    const ENSWrapper = await deployer.deploy(ENS);
+    const ENSWrapper = await deployer.deploy(ENSRegistryWithFallback);
```

Comment:
`ENSRegistryWithFallback` takes an argument in the constructor. I believe etherlime will complain if we don't explicitly pass `address(0)` but I could be wrong.

Reply:
You're right. I fixed it in the tests but not here as these aren't run during testing. How is it we verify the deployment scripts?